scripts/apps/authoring/styles/authoring.scss

```diff
@@ -212,99 +212,145 @@
 // Opened articles
 // -------------------------------------------------
 
-.opened-articles {
+$bottom-bar-item-border-color: rgba(255,255,255,0.1);
+$bottom-bar-item-border-color-active: transparent;
+
+.opened-articles-bar {
     position: fixed;
-    min-height: $authoring-opened-articles;
-    left: 0; right: 0;
+    height: $authoring-opened-articles;
+    left: 0; 
+    right: 0;
     bottom: 0;
-    background: rgba(0,0,0,0.85);
+    background: rgba(20, 20, 20, 0.95);
     z-index: $zindexFixedNavbar;
-
-    .opened-articles__list > li {
-        @include border-box();
-        padding: 0 1.6rem;
-        max-width: 24rem;
+    white-space: nowrap;
+    max-width: 100vw;
+    display: grid;
```

Comment:
I hade problems making it behave like intended in all cases with flex. Grid produced better results when there is a lot of items in the list. And how simpler would it really be?

Reply:
For one, flexbox is older thus more common and more people know how to use it. For example, I never took the time to learn how CSS grids work since I've never had a problem that couldn't be solved with flexbox.

From my initial impressions, grid also seems to be less flexible.

> grid-template-columns: 48px 1fr auto auto;

For example, here the container is coupled to its children. If I wanted to add another button or two, I'd need to also update the widths in container styles or add more columns. When using flexbox, you define which parts grow and which shrink, but the width of children doesn't need to be specified and adjust automatically. See https://jsfiddle.net/9n6sbuto/

I'm wondering which cases were hard to make behave correctly?

Reply:
> For one, flexbox is older thus more common and more people know how to use it. For example, I never took the time to learn how CSS grids work since I've never had a problem that couldn't be solved with flexbox.

There are a quite a few of things grid does that flexbox doesn't. You should take the time as it is fully supported by all major browsers. As an example what it does that flex doesn't take a look at the Photo grid view in Superdesk. 

> For example, here the container is coupled to its children. If I wanted to add another button or two, I'd need to also update the widths in container styles or add more columns. When using flexbox, you define which parts grow and which shrink, but the width of children doesn't need to be specified and adjust automatically. See https://jsfiddle.net/9n6sbuto/

No, you don't have to specify any width. 48px is reserved for the button on the far left, which should always be the same width as the sidebar. The rest is flexible, 1fr stretches the item and auto adapts so it can fit more items inside. The example you provided is pretty much how it was before and didn't behave as desired. 

> I'm wondering which cases were hard to make behave correctly?
With a large number of open articles, the list started to push the Create button outside the viewport. With grid, it was not the case. 

But let's not turn this into a long debate – honestly, it could be just my personal preference and it would still be a perfectly valid way to do it.